webapp/channels/src/components/channel_header/channel_header.tsx

```diff
@@ -528,7 +528,7 @@ class ChannelHeader extends React.PureComponent<Props, State> {
                     id='header-popover'
                     popoverStyle='info'
                     popoverSize='lg'
-                    style={{transform: `translate(${this.state.leftOffset}px, ${this.state.topOffset}px)`}}
+                    style={{transform: `translate(${this.state.leftOffset}px, ${this.state.topOffset}px)`, maxWidth: 'calc(100% - 400px)'}}
```

Comment:
I apologize for the huge delay. 
Unfortunately, I could not make it work only using classnames in css files, because in this case we need to know the width of the parent component and if there are multiple teams. That we can know only using JS. 
I fixed the above mentioned scenarios. Can you please check again @hmhealey ?

Reply:
No worries at all about the delay. There's absolutely no rush on this.

That makes sense doing this through JS then